src/crate/client/sqlalchemy/tests/insert_from_select_test.py

```diff
@@ -66,17 +68,41 @@ class CharacterArchive(Base):
         self.character_archived = CharacterArchive
         self.session = Session()
 
+    @unittest.skipIf(SA_VERSION >= SA_1_4, "Not supported on SQLAlchemy >= 1.4")
     @patch('crate.client.connection.Cursor', FakeCursor)
-    def test_insert_from_select_triggered(self):
+    def test_insert_from_select_triggered_1_0_0(self):
```

Comment:
```suggestion
    def test_insert_from_select_on_1_0_0_contains_parenthesis(self):
```

Reply:
Thanks. I've adjusted the function name in the original commit.

Reply:
This also became obsolete with #424.